model_analyzer/entrypoint.py

```diff
@@ -93,12 +97,21 @@ def get_server_handle(args):
     elif args.triton_launch_mode == 'docker':
         triton_config = TritonServerConfig()
         triton_config['model-repository'] = args.model_repository
+        http_port = args.triton_http_endpoint.split(':')[-1]
```

Comment:
You can just load them into the server config right? Do you need the dict here?

Reply:
Yes, loading it there will lead to changing the `tritonserver` port inside the container which is not what we want.

Reply:
Can we not change the `tritonserver` port to match free ports?

Reply:
You're right I can just change the server config and use the same values for both inside and outside the container.